src/components/ListItem.jsx

```diff
@@ -63,6 +69,7 @@ export function ListItem({
 			<label className="ListItem-label" htmlFor={name}>
 				{name}
 			</label>
+			<button onClick={handleDelete}>Delete</button>
```

Comment:
Nice and clean! Only thing I'd recommend for the buttons is adding `type=button` generally for the new elements.

Reply:
resolved!